src/main/kotlin/uk/gov/gdx/datashare/enums/EnrichmentField.kt

```diff
@@ -20,4 +20,5 @@ enum class EnrichmentField(@JsonValue val jsonName: String) {
   PRISONER_NUMBER("prisonerNumber"),
   FIRST_NAME("firstName"),
   MIDDLE_NAMES("middleNames"),
+  GENDER("gender"),
```

Comment:
Why do we have 2 types of sex/gender ?  Is this because the two models have different results?

Reply:
The reasoning is that they're fundamentally different things, that happen to have the same shape. I think we need to be very clear on them being different to our downstream acquirers.